# Request
string port
string msg
---
# Response
bool success
string message
